.ci/build.sh

```diff
@@ -9,14 +9,16 @@ mkdir private _site public
 
 META_REVIEW_DATA="meta_review.json"
 
-EXPORTED_DATA="static/tasks.yaml static/instances.yaml static/$META_REVIEW_DATA"
+EXPORTED_DATA="static/tasks.yaml static/instances.yaml static/$META_REVIEW_DATA\
```

Comment:
LineLengthBear givers error that's why I split this up.Is this the correct way to do this? Expect the indentation part I'll do that.